### DISCUSSION TITLE
List of GSoC ideas updated

### DISCUSSION THREAD

**Discussion by romain-fontugne** [InternetHealthReport/gsoc#37] [Announcements]
We have updated our list of ideas for GSoC'24! (although IHR is not yet officially part of GSoC'24 :crossed_fingers: )
https://github.com/InternetHealthReport/gsoc/blob/main/ideas.md

Please use github discussions to get clarifications about the projects.
All instructions for contributing to IHR are available here:
https://github.com/InternetHealthReport/gsoc/

**Comment by MAVRICK-1** [comment]
@romain-fontugne I want a propose an IDEA on #16 with some further implementation

**Reply by romain-fontugne** [reply]
No problem, you can propose projects that are not in the list of ideas. But given the load of work we have, we are not going to review proposals before the official selection of GSoC organizations.

**Reply by MAVRICK-1** [reply]
> No problem, you can propose projects that are not in the list of ideas. But given the load of work we have, we are not going to review proposals before the official selection of GSoC organizations.

@romain-fontugne okay sir :-).I already created a working demo . I will include it in the proposal after official selection of Gsoc organization.🤞
